razorpay/utility/utility.py

```diff
@@ -20,7 +20,21 @@ def verify_payment_signature(self, parameters):
         secret = str(self.client.auth[1])
 
         return self.verify_signature(msg, razorpay_signature, secret)
+    
+    def verify_subscription_payment_signature(self, parameters):
+        """
+        To consider the payment as successful and subscription as authorized 
+        after the signature has been successfully verified
+        """
+        subscription_id = str(parameters['razorpay_subscription_id'])
+        payment_id = str(parameters['razorpay_payment_id'])
+        razorpay_signature = str(parameters['razorpay_signature'])
 
+        msg = "{}|{}".format(payment_id, subscription_id)
+        secret = str(self.client.auth[1])
```

Comment:
@sanzzzay The above questions applies here as well, this time the index `1` in `self.client.auth`

Reply:
I have written a utility method in my code `get_client` where I am passing API_KEY and API_SECRET and handling the exception so won't encounter this issue.